jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/ButtonToggle.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.TabsAssert;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+
+public class ButtonToggle extends UIBaseElement<TabsAssert> {
+
+    public void clickButtonByValue(String value){
+        getButtonByTagValue(value).click();
+    }
+
```

Comment:
In my opinion, all public methods of the element that you will use in tests should contain an annotation @JDIAction with corresponding description of.